Logger/FileLogger.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        public string? FilePath { get; set; }
+        //string? ClassName inherited from BaseLogger
+        public FileLogger(string className, string filePath)
+        {
+            ClassName = className;
+            FilePath = filePath;
+        }
+
+        public override void Log(LogLevel logLevel, string message)
+        {
+            //example: 10/7/2019 12:38:59 AM FileLoggerTests Warning: Test message
+            File.AppendAllText(FilePath, $"{DateTime.Now} {ClassName} {logLevel}: {message} \n");
```

Comment:
```suggestion
            File.AppendAllText(FilePath, $"{DateTime.Now} {ClassName} {logLevel}: {message}{Environment.NewLine}");
```
Looks great, my only suggestion would be to use `Environment.NewLine` instead of `\n`